pkg/controller/pd_control.go

```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/pingcap/tidb-operator/pkg/pdapi"
 )
 
-// GetPDClient gets the pd client from the TidbCluster
-func GetPDClient(pdControl pdapi.PDControlInterface, tc *v1alpha1.TidbCluster) pdapi.PDClient {
+// GetPDClientBasic gets the pd client from the TidbCluster
+func GetPDClientBasic(pdControl pdapi.PDControlInterface, tc *v1alpha1.TidbCluster) pdapi.PDClient {
```

Comment:
```suggestion
// getPDClientInCurrentK8s gets the pd client from the TidbCluster
func getPDClientInCurrentK8s(pdControl pdapi.PDControlInterface, tc *v1alpha1.TidbCluster) pdapi.PDClient {
```
should this be a private function?

Reply:
Yes, I will change it.